lib/core/themes/text_styles.dart

```diff
@@ -7,46 +7,55 @@ class TextStyles {
     fontSize: 10.sp,
     fontWeight: FontWeight.w700,
     color: AppColors.white,
+    fontFamily: 'SpaceGrotesk',
   );
   static TextStyle textstyle12 = TextStyle(
```

Comment:
what's ( textstyle12 ) ?

it's better if we name it like ( font12White700w ) or ( font13DarkBlueMedium ) any similar descriptive name.

Reply:
done